### PR TITLE
load env configs for dbms connectivity

### PR DESCRIPTION
**Pull request by singhalyash8080**
I integrated dotenv to use environment variables for dbms connectivity and also I regenerated package-lock.json which would be disturbing others after cloning repo for running it.

### CODE REVIEW

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
1. your commit messages don't have a github user as author, please fix that.
2. add these entries to `.env.example`

**Comment by singhalyash8080** [comment]
added github username in commit message and added .env entries to .env.example

**Comment by singhalyash8080** [comment]
I have done changes . Please review .

File: .env.example [hunk 1]
```diff
@@ -1 +1,3 @@
-PORT=3000
```
Comment (YashKumarVerma) [inline .env.example:1]: why is this variable removed ? please don't.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (singhalyash8080) [reply .env.example:1]: Oh sorry , I will fix that

Comment (singhalyash8080) [reply .env.example:1]: solved that problem, please review.

File: src/database.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+
 
 var con = mysql.createConnection({
-  host: "localhost",
-  user: "yourusername",
-  password: "yourpassword"
+  host: process.env.DB_HOST || 'localhost',
+  user: process.env.DB_USER || 'yourUser',
+  password: process.env.DB_PASS || 'yourPass',
+  port: process.env.DB_PORT || 3000
```
Comment (YashKumarVerma) [inline src/database.js:10]: this isn't the default mysql port, but i'll approve it :smile:

**Review by YashKumarVerma — APPROVED** [review]

**Comment by lopeselio** [comment]
There are merge conflicts. Need to resolve.

**Comment by lopeselio** [comment]
Need to resolve merge conflicts on `src/database.js`

**Comment by singhalyash8080** [comment]
resolved conflicts, you can check again

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
Thank you !

**Pull request merged**